### PR TITLE
Create CSS Values and Units.md

### LINKED ISSUE #1158: CSS values and units
<hr>

## Description 📜

This documentation will contain information about units in css.

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev (HTML, CSS, JS)

<hr>

### PR DESCRIPTION
**Pull request by Pranavanbu28**
<hr>

## Description 📜

This documentation information about CSS Values and Units.
Fixes #1158

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Pranavanbu28** [comment]
@code-reaper08 @praveenscience Please do check my contribution.

**Review by code-reaper08 — APPROVED** [review]
Awesome @Pranavanbu28 👏
Looks fine to me.

**Comment by Pranavanbu28** [comment]
@praveenscience  should I change something?

File: Web_Development/FrontEnd/CSS Values and Units.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<hr>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS Values and Units.md:1]: Remove this.

File: Web_Development/FrontEnd/CSS Values and Units.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+## CSS Properties and Values
+Now before we get into units, we must have some basic knowledge about what *properties* are. The CSS Properties are used to modify and customise the standard HTML elements like divs, paragraphs, sections and a whole lot more according to our own free will. CSS declarations work in a property / value pair. In short a property is **what you want to modify** in a page and the value you set tells you **how you want to modify it**. Example:
+
+```
+.redbox{
+  height:100px;
+  width:100px;
+  background-color:red;
+}
+```
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS Values and Units.md:14]: Add syntax, format the code with spaces.

File: Web_Development/FrontEnd/CSS Values and Units.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+```
+ .cat-image{
+  height:20vmin; // takes up 20% of the screen's minimum dimension as its height
+  width:10vmax // takes up 10% of the screen's maximum dimension as its width
+ }
+```
+Thanks for reading !
+
+<hr>
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS Values and Units.md:103]: Remove

File: Web_Development/FrontEnd/CSS Values and Units.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+}
+```
+- vmin and vmax are used when an element has to take up its dimensions relative to the the screen's smaller dimension or the larger dimension.
+
+```
+ .cat-image{
+  height:20vmin; // takes up 20% of the screen's minimum dimension as its height
+  width:10vmax // takes up 10% of the screen's maximum dimension as its width
+ }
+```
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS Values and Units.md:100]: Add CSS syntax for all these.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Can you please fix these.

File: Web_Development/FrontEnd/CSS Values and Units.md (hunk 4 above)
Comment (Pranavanbu28) [reply Web_Development/FrontEnd/CSS Values and Units.md:100]: sir syntax in the sense like 20em like that?

**Review by praveenscience — APPROVED** [review]

**Pull request merged**